tests/test_regression.py

```diff
@@ -737,6 +737,15 @@ def test_nested_loop_scoping(self, env):
         assert tmpl.render() == "hellohellohello"
 
 
+def test_load_parameter_when_set_in_all_if_branches(env):
+    tmpl = env.from_string(
+        "{% if True %}{{ a.b }}{% set a = 1 %}"
+        "{% elif False %}{% set a = 2 %}"
+        "{% else %}{% set a = 3 %}{% endif %}"
+    )
+    assert tmpl.render(a={"b": 0}) == "0"
```

Comment:
This should probably output `{{ a }}` after the `{% endif %}` and test that both the before and after are what we expect.